CHANGELOG.md

```diff
@@ -32,19 +46,42 @@
     where `LISTEN_INTERFACE` is the value from the `--listen-interface` option if specified, or `127.0.0.1` otherwise)
   - See [Linux Installer Script](https://github.com/signalfx/splunk-otel-collector/blob/main/docs/getting-started/linux-installer.md)
     for more details.
-- (Splunk) Add the loadbalancing exporter to the distribution ([#3825](https://github.com/signalfx/splunk-otel-collector/pull/3825))
-- (Splunk) Add UDP log receiver ([#3826](https://github.com/signalfx/splunk-otel-collector/pull/3826))
+- (Splunk) Update splunk-otel-javaagent to [v1.29.0](https://github.com/signalfx/splunk-otel-java/releases/tag/v1.29.0) ([#3788](https://github.com/signalfx/splunk-otel-collector/pull/3788))
+- (Splunk) Redis discovery ([#3731](https://github.com/signalfx/splunk-otel-collector/pull/3731))
+- (Splunk) Update Bundled OpenJDK to [11.0.21+9](https://github.com/adoptium/temurin11-binaries/releases/tag/jdk-11.0.21%2B9) ([#3819](https://github.com/signalfx/splunk-otel-collector/pull/3819))
+- (Splunk) Oracledb discovery tweaks (remove static endpoint) ([#3836](https://github.com/signalfx/splunk-otel-collector/pull/3836))
+- (Contrib) `probabilisticsamplerprocessor`: Allow non-bytes values to be used as the source for the sampling decision ([#18222](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/18222))
+- (Contrib) `kafkareceiver`: Allow users to attach kafka header metadata with the log/metric/trace record in the pipeline. Introduce a new config param, 'header_extraction' and some examples. ([#24367](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/24367))
+- (Contrib) `kafkaexporter`: Adding Zipkin encoding option for traces to kafkaexporter ([#21102](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/21102))
+- (Contrib) `kubeletstatsreceiver`: Support specifying context for `kubeConfig` `auth_type` ([#26665](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/26665))
+- (Contrib) `kubeletstatsreceiver`: Adds new `k8s.pod.cpu_limit_utilization`, `k8s.pod.cpu_request_utilization`, `k8s.container.cpu_limit_utilization`, and `k8s.container.cpu_request_utilization` metrics that represent the ratio of cpu used vs set limits and requests. ([#27276](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/27276))
+- (Contrib) `kubeletstatsreceiver`: Adds new `k8s.pod.memory_limit_utilization`, `k8s.pod.memory_request_utilization`, `k8s.container.memory_limit_utilization`, and `k8s.container.memory_request_utilization` metrics that represent the ratio of memory used vs set limits and requests. ([#25894](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/25894))
+- (Core) `service/telemetry exporter/exporterhelper`: Enable sampling logging by default and apply it to all components. ([#8134](https://github.com/open-telemetry/opentelemetry-collector/pull/8134]))
```

Comment:
I'd consider this a breaking change even if not advertised as such by their release notes.